.github/workflows/scripts/eslint_disable_check.py

```
@@ -0,0 +1,155 @@
+        3) Pycodestyle
+        4) Flake8
+        5) Python Black
+
+"""
+import os
+import re
+import argparse
+import sys
+
+
+def has_eslint_disable(file_path):
```

Ruff (0.8.2)

Missing return type annotation for public function `has_eslint_disable`

(ANN201)